src/main/java/trackr/model/util/SampleDataUtil.java

```diff
@@ -90,6 +96,22 @@ public static ReadOnlyTaskList getSampleTaskList() {
         return sampleTl;
     }
 
+    public static MenuItem[] getSampleMenuItems() {
+        return new MenuItem[] {
+            new MenuItem(new ItemName("Chocolate Cookies"), new ItemPrice("5"), new ItemCost("1")),
```

Comment:
Following the regex, can change to `5.00`, `1.00` etc.

Should also make this the command format of the parameter

Reply:
Thank you I will implement after this PR merges